.github/workflows/welcome.yml

```diff
@@ -0,0 +1,18 @@
+name: Send a Welcome Message to First time contributors
+on:
+  pull_request_target:
+    types: [opened]
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download Client
+        run: curl -#L $CLI_URL --output fbot.jar
+        env:
+          CLI_URL: ${{ secrets.BOTJAR_URL }}
+      - name: Send Welcome Message
+        run: java -jar fbot.jar welcomemsg
```

Comment:
Where do I see this `welcomemsg`?

Reply:
Infact that is embedded in the JAR client.

Reply:
Link to the body pls, need to see it

Reply:
``` java
public class Messages {
    public static final String WELCOME_MESSAGE="Thanks for taking your time out to contribute to this list ! \nWe hope you've read the [Contributing Guidelines](../blob/master/CONTRIBUTING.MD) and followed them. Your pull request will be reviewed by a reviewer soon.";
    public static final String LINT_MESSAGE_SUCCESS="Lint passed";
    public static final String LINT_MESSAGE_FAILURE="Lint Failed";
    public static final String PREVIEW_DEPLOYMENT="Deployed to surge.sh";
    public static final String UNPRIVILEGED_AUTHOR="You are not allowed to modify this file";
    public static final String ASK_AUTHOR_TOCHK="@{pr_author} please review the deployment and reply with `@freefordevbot done` when you have self-reviewed the deployment";
    public static final String MERGE_MSG="Thanks for contributing ! \nYour Changes are live now [here](https://free-for.dev).";
}
```

Other messages have not been implemented yet.

@ripienaar